modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -33,6 +35,22 @@
 public enum AndesConfiguration implements ConfigurationProperty {
 
     /**
+     * The coordination type which will be used to synchronize cluster events. If coordination type is set to
+     * "HAZELCAST" cluster communication will be handled through Hazelcast messages where as if coordination type is
+     * set to "RDBMS", it'll be via the database
+     */
+    CLUSTER_EVENT_SYNC_MODE("coordination/clusterEventSyncMode", ClusterEventSyncMode.RDBMS.toString(),
+            ClusterEventSyncMode.class),
+
+    /**
+     * The interval at which, the cluster events will be read from the database. Specified in seconds. This will only
+     * be applied if 'clusterEventSyncMode' is set to 'RDBMS'. Setting this to a very low value could have an impact
+     * on the performance since the number of DB calls will be high. Setting this with a large value increases the
+     * time taken for a cluster event to be synchronized.
+     */
+    CLUSTER_EVENT_SYNC_INTERVAL("coordination/clusterEventSyncInterval", "1", Integer.class),
```

Comment:
Since this apply only in RDBMS mode, better to make it nested under sync mode. 

May be something similar to,
```
<coordination>
  <clusterEventSyncMode>
    <RDBMS enabled="true">
      <syncInterval>1</syncInterval>
    </RDBMS>
    <!--hazelcast enabled="false">
    </hazelcast-->
  </clusterEventSyncMode>
<coordination>
```